src/nxdoc/nuxeo-add-ons/nuxeo-aspera-connector.md

```diff
@@ -14,15 +14,388 @@ tree_item_index: 1100
+
+#### Transfer transition types
+Transitioning between the “upload” and “create” steps can be achieved in 2 ways: **automatically (auto-create)** or **manually**. Users can choose which option to use when creating the transfer from the transfer dashboard.   It is possible to change from manual to auto-create after you start adding content to the transfer.  **Once you set the to auto-create, however, the transfer will behave as if you started in that way, and some limitations around modifying the transfer will apply**.
+
+When auto-create is toggled/selected to be “on”, the user needs only to start the upload of the folders/files to the transfer.  Once the upload is completed, the system will automatically start the process to create the content in the system; no need to wait around simply to push a button.  **Before adding content** to the transfer, you can edit the **“common metadata”** which would apply to all files in the transfer.  Users can, of course, decide not to include any metadata at all, and simply add files, too.
+
+{{#> callout type='note' }}
+There are some caveats when using auto-create (auto-create is set to “on”).  Because of some asynchronous behavior, the ability to modify the transfer/file metadata or to add/remove files once your initial "add to transfer" has begun, is not allowed.  If you discover that you have erred in the files/information, you will need to let the transfer process complete before addressing within the system; where the content was added (not the transfer).
+{{/callout}}
+
+If a user has content that needs to be “tagged” or “catalogued” with metadata before creation in the system, the “manual” transition option might be a better choice. To use the **manual option**, choose “no” for auto-create or toggle the auto-create to “off” when creating a new transfer (you can only turn off auto-create if there is **no content in the transfer)**.  This will require someone **manually start** the creation process by clicking the **“complete transfer”** button once the upload is finished (the button is only visible once there is no longer an upload in progress)
+When using the manual transition, the user uploads content to the transfer bucket. While in this step, a user can add folders and files to the bucket and add properties that apply to the files. Properties can be added that apply to all files using the “common metadata” panel; more file-specific properties can be added either to multiple files at once (bulk edit of metadata) and/or individually,  using a single-file action.
+Once the metadata has been edited to fit the user's needs, the transfer can be **completed** and the content is then created in the system.
+Transfers have 3 states: Draft (no content added yet), In Progress (content added) and Completed (the transfer has been completed and the content has been created in the system).
```

Comment:
Indeed, in visual terms! Because technically we have one more state on the lifecycle but I guess it is not important to be stated here.

Reply:
@nmpcunha more or less, **but** I don't believe we show the "draft" state anymore.  we only show a status icon for started/pending/completed.  showing the draft state was too confusing. this is the draft state (and on the dash, we just have the actions)
![Screen Shot 2020-10-13 at 3 21 46 PM](https://user-images.githubusercontent.com/18596875/95912075-f188d080-0d67-11eb-8fd5-fb6047990be3.png)

Reply:
Indeed, there is nothing right now stating "Draft", which may be also tricky from a functional perspective to state that.
Anyway, what does make sense to state as `states` from a "functional perspective"?

Reply:
I am wondering if we can summarise those scenarios in a given table,

it's mainly a suggestion / proposal but we can live with the description :)

Reply:
yes, will remove the reference to the "draft" state (since we don't visualize it) and focus on what we do see (pending/started/completed for now, I believe)..  I think once we get our LC states redone for transfers, then a table will be the best way to summarize and can make it similar in "function" to the sheet we have on our wiki page and include manual/auto create info, too.